src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ClimbingPick.java

```diff
@@ -235,7 +235,7 @@ private void swing(Player p, Block b, EquipmentSlot hand, ItemStack item) {
     }
 
     @Override
-    public void damageItem(Player p, ItemStack item) {
+    public void damageItem(@Nonnull Player p, ItemStack item) {
```

Comment:
nullability for `item` too

Reply:
`item` is `Nullable` according to `DamageableItem.java` at line `48`